Makefile

```
@@ -266,6 +266,9 @@ constraints.txt: $(REQUIREMENTS)
 	.dataflow-versions.txt \
 	$^ \
 	--output-file constraints.txt
+	# remove constraints from GitHub, since pip does not support unnamed
```

```suggestion
	# remove GitHub links from constraints, since pip does not support unnamed
```
is this what is meant?

---

Yup, that's clearer.  Thanks!